src/f32/vec4_f32.rs

```diff
@@ -573,21 +578,38 @@ impl Vec4Mask {
         self.bitmask()
     }
 
+    /// Returns a bitmask with the lowest four bits set from the elements of
```

Comment:
It might be worth mentioning that only the most significant bit of each lane (element) is used for creating the bitmask. I'm using the packed_simd docs for inspiration here https://rust-lang-nursery.github.io/packed_simd/packed_simd/type.m32x4.html#method.bitmask

Reply:
Same question here as above.

Reply:
I guess to expand on that: is this crate planning to only support x86/64 SIMD into the future?  I expect that specifics like this would vary from platform to platform.

Reply:
I'd like to support other architectures at some point. I'm working on the assumption that this holds across different architectures because `packed_simd` supports multiple architectures. That said, `packed_simd` is an experimental crate and might not be correct.